source/MaterialXView/Image/TinyEXRImageHandler.cpp

```diff
@@ -45,16 +45,57 @@ bool TinyEXRImageHandler::saveImage(const std::string& fileName,
     }
     return (returnValue == 0);
 }
-#else
-    bool TinyEXRImageHandler::saveImage(const std::string& /*fileName*/,
-        const std::string& /*extension*/,
-        unsigned int /*width*/,
-        unsigned int /*height*/,
-        unsigned int /*channelCount*/,
-        const float* /*buffer*/)
+
+bool TinyEXRImageHandler::loadImage(const std::string& fileName,
+                                    const std::string& extension,
```

Comment:
Would it be better to read the extension from the given file name instead?

Reply:
I can add this in. I may add in a "is supported codec" later as a separate interface.